src/ConnectedMachine/readme.md

```diff
@@ -0,0 +1,74 @@
+<!-- region Generated -->
+# Az.ConnectedMachine
+This directory contains the PowerShell module for the ConnectedMachine service.
+
+---
+## Status
+[![Az.ConnectedMachine](https://img.shields.io/powershellgallery/v/Az.ConnectedMachine.svg?style=flat-square&label=Az.ConnectedMachine "Az.ConnectedMachine")](https://www.powershellgallery.com/packages/Az.ConnectedMachine/)
+
+## Info
+- Modifiable: yes
+- Generated: all
+- Committed: yes
+- Packaged: yes
+
+---
+## Detail
+This module was primarily generated via [AutoRest](https://github.com/Azure/autorest) using the [PowerShell](https://github.com/Azure/autorest.powershell) extension.
+
+## Module Requirements
+- [Az.Accounts module](https://www.powershellgallery.com/packages/Az.Accounts/), version 1.6.0 or greater
+
+## Authentication
+AutoRest does not generate authentication code for the module. Authentication is handled via Az.Accounts by altering the HTTP payload before it is sent.
+
+## Development
+For information on how to develop for `Az.ConnectedMachine`, see [how-to.md](how-to.md).
+<!-- endregion -->
+
+<!-- region Generated -->
+# Az.ConnectedMachine
+This directory contains the PowerShell module for Hybrid Compute.
+
+---
+## Run Generation
+In this directory, run AutoRest:
+> `autorest`
+
+---
+### AutoRest Configuration
+> see https://aka.ms/autorest
+
+``` yaml
+use: "@microsoft.azure/autorest.powershell@beta"
```

Comment:
These configuration file need some changes. Please use https://github.com/Azure/azure-powershell/blob/generation/src/Functions/readme.md as an example. And Highlight some things as below.
1. Instead of using input-file, profile should be used instead.
2. Common configuration is put in ../readme.azure.md, and you just need require this file in your readme.md